packages/signature-controller/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/signature-controller",
-  "version": "21.0.0",
+  "version": "22.0.0",
```

Comment:
Since our new constructor option isn't required, this isn't a breaking change so we can just do a minor version.

Reply:
Fixed.